packages-exp/auth-exp/src/core/user/user_impl.test.ts

```diff
@@ -116,9 +128,22 @@ describe('core/user/user_impl', () => {
   });
 
   describe('#delete', () => {
-    it('throws', () => {
+    it('calls delete endpoint', async () => {
+      stsTokenManager.updateFromServerResponse({
+        idToken: 'id-token',
+        refreshToken: 'refresh-token-string',
+        expiresIn: '100000'
+      } as IdTokenResponse);
       const user = new UserImpl({ uid: 'uid', auth, stsTokenManager });
-      expect(() => user.delete()).to.throw();
+      const endpoint = mockEndpoint(Endpoint.DELETE_ACCOUNT, {});
+      const signOut = sinon.stub(auth, 'signOut');
```

Comment:
is it possible to test side effects instead of mocking this out?

Reply:
Will do this once https://github.com/firebase/firebase-js-sdk/pull/2963 merges